js/app.jsx

```diff
@@ -159,6 +187,106 @@ var SearchableRemoteListMixin = {
 
 }
 
+var ListToolbar = React.createClass({
```

Comment:
I missed the content of this, seem like different lists have totally different set of buttons, how about making it `<CaseverListToolbar/>` and `<SuiteListToolbar/>`, just like `<*ListItem/>`?